source/windows/iocp/iocp_event_loop.c

```diff
@@ -414,8 +416,8 @@ static void s_schedule_task_common(struct aws_event_loop *event_loop, struct aws
         AWS_LOGF_TRACE(
             AWS_LS_IO_EVENT_LOOP,
             "id=%p: scheduling task %p in-thread for timestamp %llu",
-            event_loop,
-            task,
+            (void *)event_loop,
+            (void *)task,
             (unsigned long long)run_at_nanos);
```

Comment:
Instead of this cast, you can use format specifier `%zu` for `size_t` and `%zd` for `ssize_t`.

Reply:
zu and zd aren't supported on sufficiently old versions of msvc